VERSION

```diff
@@ -2,4 +2,4 @@ VERSION_MAJOR = 2
 VERSION_MINOR = 2
 PATCHLEVEL = 0
 VERSION_TWEAK = 0
-EXTRAVERSION = rc3
+EXTRAVERSION =
```

Comment:
I tihnk you need a separate PR for this change.

Reply:
I was just  following what what was done for 2.1 with commit b413223a66ed9e857c67ee21d1e8be4768804dd1

Reply:
Anyway, why would a separate PR be needed?

Reply:
Well, actually, a separate commit (but then a separate PR would also make CI run once more).
You should update the version file with a commit with "release: " in the title, and appropriate content.
https://github.com/zephyrproject-rtos/zephyr/wiki/Release-procedure